src/mlpack/core/data/csv_parser.hpp

```diff
@@ -0,0 +1,97 @@
+/** 
+ * @file core/data/csv_parser.hpp
+ * @author Gopi M. Tatiraju
+ *
+ * This csv parser is designed by taking reference from armadillo's csv parser.
```

Comment:
You need to add your name in addition to Conrad's name, not to replace it with your name. Since the code would not have been existed without his work

Reply:
Actually in armadillo authors are not mentioned explicitly as in mlpack, so I copied the contents from there as it is and added my name. I will add cornads name as well there.